types.go

```diff
@@ -229,47 +230,36 @@ func convertZvalToGo(zval *C.zval) interface{} {
 
 // convertGoToZval converts a Go interface{} to a PHP zval
 func convertGoToZval(value interface{}) *C.zval {
-	zval := (*C.zval)(C.__emalloc__(C.size_t(unsafe.Sizeof(C.zval{}))))
-	u1 := (*C.uint8_t)(unsafe.Pointer(&zval.u1[0]))
-	v0 := unsafe.Pointer(&zval.value[0])
+	zval := C.__zval_alloc__()
```

Comment:
```suggestion
	var zval C.zval
```

You don't need to allocate here. In order to avoid leaking, you just need to declare the zval and then pass a reference, PHP will handle allocation

```c
switch v := value.(type) {
	case nil:
		C.__zval_null__(&zval)
	case bool:
		C.__zval_bool__(&zval, C._Bool(v))
	case int:
		C.__zval_long__(&zval, C.zend_long(v))
        case ...
```

When running PHP in debug mode (built with --enable-debug like in the [dev.Dockerfile](https://github.com/php/frankenphp/blob/main/dev.Dockerfile)) you should see something like this when there is a leak between:

```
Last leak repeated 2 times
=== Total 3 memory leaks detected ===
[Fri Aug  1 21:58:55 2025]  Script:  '-'
```